samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable react/jsx-no-bind */
+import {handle, forward} from '@enact/core/handle';
+import kind from '@enact/core/kind';
+import {clamp} from '@enact/core/util';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, select} from '@enact/storybook-utils/addons/knobs';
+import Routable, {Route, Linkable} from '@enact/ui/Routable';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Button from '@enact/agate/Button';
+import Header from '@enact/agate/Header';
+import Icon from '@enact/agate/Icon';
+import Item from '@enact/agate/Item';
+import {Panels, Panel, BreadcrumbPanels} from '@enact/agate/Panels';
+import Scroller from '@enact/agate/Scroller';
+
+import componentCss from './MenuItem.module.less';
+
+Panels.displayName = 'Panels';
+const Config = mergeComponentMetadata('Panels', Panels);
+
+const BasicPanels = () => {
+	const [index, setIndex] = React.useState(0);
+	const goNext = () => setIndex(clamp(0, 2, index + 1));
+	const goPrevious = () => setIndex(clamp(0, 2, index - 1));
+	const goFirst = () => setIndex(0);
+	const goThird = () => setIndex(2);
+
+	return (
+		<Panels
+			index={index}
+			noAnimation={boolean('noAnimation', Config, false)}
+			noCloseButton={boolean('noCloseButton', Config, false)}
+			orientation={select('orientation', ['horizontal', 'vertical'], Config)}
+		>
+			<Panel>
+				<Header title="First Panel" />
+				<Item onClick={goNext}>Second</Item>
+				<Item onClick={goThird}>Third</Item>
```

Comment:
It could be my personal subjective opinion, but it looks complicated as the app is in a circular structure.
In particular, it looks that the focus doesn't preserve when I go to a below path.
First -> Third -> Second -> Third (the focus is not on "second")

Wouldn't it be better to replace the text with "previous" and "next", and get rid of moving the two panels?

Reply:
Per my understanding, getting rid of moving panels means disabling the animation, right?
Also, I tried to fix the focus on the third panel, but couldn't do it, I think it may be a bug with the Panels.

Reply:
> Per my understanding, getting rid of moving panels means disabling the animation, right?

No, I meant to move two panels at once.  (eg first->third, third->first)

Reply:
Ok, I understand and I made the changes